docs/en/observability/analyze-monitors.asciidoc

```diff
@@ -19,17 +19,17 @@ image::images/uptime-status-panel.png[Uptime status panel]
 
 The *Monitoring from* list displays service availability per monitoring location,
 along with the amount of time elapsed since data was received from that location.
-The availability percentage is the percentage of successful checks made during
+The availability percentage is the percentage of successful checks that were made during
 the selected time period.
 
-To display a map with each location as a pinpoint, you can toggle the availability view from list
-view to map view.
+Toggle the availability view from list
+view to map view to display a map with each location as a pinpoint.
```

Comment:
I would keep the original. It's good to explain to a user what they can do and then tell them how they can do it.